### PR TITLE
#133842279 Accounts Tests Fixing

### PR DESCRIPTION
**Pull request by ishiundu**
#### What does this PR do?
It contains test fixes for the `accounts/tests`

#### Description of Task to be completed?
Have the following tests working
`hc/accounts/tests/test/test_check_token` 
`hc/accounts/tests/test/test_login`
`hc/accounts/tests/test/test_profile`
`hc/accounts/tests/test/test_switch_team`
`hc/accounts/tests/test/test_team_access_middleware`

#### How should this be manually tested?
After cloning the repo, cd into it and RUN `./manage.py test hc.accounts.tests`

#### What are the relevant pivotal tracker stories?
#133842279

### CODE REVIEW

File: hc/accounts/tests/test_check_token.py [hunk 1]
```diff
@@ -22,7 +23,31 @@ def test_it_redirects(self):
...
+        #Change url to have an invalid token.
+        url = "/accounts/check_token/alice/incorrect-token/"
+        r = self.client.post(url)
+        self.assertNotEqual(url,"/accounts/check_token/alice/secret-token/")
+        self.assertRedirects(r, "/accounts/login/")
 
     ### Any other tests?
+
+if __name__ == '__main__':
+    unittest()
```
Comment (tonerdo) [inline hc/accounts/tests/test_check_token.py]: Why is this necessary?

File: hc/accounts/tests/test_profile.py [hunk 2]
```diff
@@ -108,3 +129,9 @@ def test_it_shows_badges(self):
         self.assertNotContains(r, "bobs-tag.svg")
 
     ### Test it creates and revokes API key
+    def test_it_revokes_api_key(self):
+        self.client.login(username="alice@example.org", password="password")
+        form = {'revoke_api_key': ''}
+        r = self.client.post("/accounts/profile/", form)
+        assert r.status_code == 200
```
Comment (tonerdo) [inline hc/accounts/tests/test_profile.py]: Use `assert.Equals`

**Review by tonerdo — CHANGES_REQUESTED** [review]
Hey guys, good job so far. Just a few changes and we should be good

File: hc/accounts/tests/test_profile.py (hunk 2 above)
Comment (ishiundu) [reply hc/accounts/tests/test_profile.py]: Noted

**Review by tonerdo — APPROVED** [review]

**Pull request merged**